docs/welcome/quickstart.mdx

```diff
@@ -18,82 +18,105 @@ import {
   Spacer,
   OrderedList,
   ListItem,
-  Image,
   Button,
 } from "@chakra-ui/react";
+import Quickstart from "@site/static/img/quickstart.jpg";
 
-# Introduction
+## 1. API keys
 
-Let's get started, first you’ll need API keys, which you can request by registering with us in the [Dashboard](https://app.tryvital.io).
+Let’s test out running Vital locally by cloning the [Quickstart app](https://github.com/tryVital/quickstart).
+You’ll need API keys, which you can receive by signing up in the [Dashboard](https://app.tryvital.io).
+Once registered, you can create a team. A team is associated with a region (either `EU` or `US`),
+the region dictates where data is stored.
 
-Once registered, you will be granted access to: the [Dashboard](https://app.tryvital.io),
-our Sandbox and Production environments, and be given a unique API key for each environment.
+You'll have two different API keys, and there are two different Vital environments.
```

Comment:
This doesn't make sense. This doesn't follow on from the previous sentence. And these seem like two different sentences. Should it not be you will get two different API Keys - one for each vital environment?